python-package/xgboost/spark/core.py

```diff
@@ -1095,12 +1095,15 @@ def _train_booster(
             context.barrier()
 
             if context.partitionId() == 0:
-                yield pd.DataFrame(
-                    data={
-                        "config": [booster.save_config()],
-                        "booster": [booster.save_raw("json").decode("utf-8")],
-                    }
-                )
+                config = booster.save_config()
+                yield pd.DataFrame({"data": [config]})
+                booster = booster.save_raw("json").decode("utf-8")
+
+                chunk_size = 4096 * 1024
```

Comment:
Could we make chunk_size to be global variable, since the below somewhere is still using it.